packages/app/src/components/SearchResultsCard/style.css

```diff
@@ -13,6 +13,33 @@
   background-color: white;
 }
 
+.search-result-span {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.search-exit {
+  width: 300%;
+  display: flex;
+  flex-grow: 1;
+  justify-content: flex-end;
+}
+
+.thead {
+  width: 300%
+}
```

Comment:
Are these being used somewhere?

Reply:
I believe they are used or deleted in Lavar and I's other ticket #137.

Reply:
If they're used in the other ticket then we should remove them from this PR and add them there instead.